examples/oft/.eslintrc.js

```diff
@@ -7,5 +7,6 @@ module.exports = {
         // @layerzerolabs/eslint-config-next defines rules for turborepo-based projects
         // that are not relevant for this particular project
         'turbo/no-undeclared-env-vars': 'off',
+        'import/no-unresolved': 'warn', // Updated to warn to avoid erratic popping up of this particular error for '@layerzerolabs/metadata-tools' even when the package has been installed correctly
```

Comment:
Why is this happening?

Reply:
I'm not sure. I tried a bunch of things - installing it as a regular dep instead of devDep, re-running `pnpm i`, then I mentioned it to @St0rmBr3w and he said he experienced it too when he was setting up the lzapp-migration example for the first time.